gradle/kotlinGradlePlugin.gradle

```diff
@@ -11,11 +11,26 @@ project.buildscript.repositories {
     maven {
         url buildKotlinCompilerRepo
     }
+    maven {
```

Comment:
Is this file used in every place we apply the Kotlin plugin?

Reply:
it's good question :)
some number:
```
0:b-compiler-update:minamoto@unit-703(1)# find . -name build.gradle | wc -l
      42
0:b-compiler-update:minamoto@unit-703(1)# find samples/ -name build.gradle | wc -l
       5
0:b-compiler-update:minamoto@unit-703(1)# git grep kotlinGradlePlugin.gradle | wc -l
      26
0:b-compiler-update:minamoto@unit-703(1)# git grep apply\ plugin:\ \'kotlin\' | wc -l
      14
```

perhaps in(or all) some of these files manner of applying of kotlin plugin should be changed

```
0:b-compiler-update:minamoto@unit-703(1)# git grep apply\ plugin:\ \'kotlin\'
Interop/Indexer/build.gradle:apply plugin: 'kotlin'
Interop/Runtime/build.gradle:apply plugin: 'kotlin'
Interop/StubGenerator/build.gradle:apply plugin: 'kotlin'
backend.native/build.gradle:apply plugin: 'kotlin'
backend.native/debugger-tests/build.gradle:apply plugin: 'kotlin'
buildSrc/plugins/build.gradle:apply plugin: 'kotlin'
buildSrc/plugins/build.gradle:apply plugin: 'kotlin'
buildSrc/shared/build.gradle:apply plugin: 'kotlin'
extracted/konan.serializer/build.gradle:apply plugin: 'kotlin'
klib/build.gradle:apply plugin: 'kotlin'
shared/build.gradle:apply plugin: 'kotlin'
shared/buildSrc/build.gradle:apply plugin: 'kotlin'
tools/kotlin-native-gradle-plugin/build.gradle:apply plugin: 'kotlin'
utilities/build.gradle:apply plugin: 'kotlin'
```

Reply:
yes